server/adapters/bigquery.go

```diff
@@ -197,6 +212,39 @@ func (bq *BigQuery) Close() error {
 	return bq.client.Close()
 }
 
+func (bq *BigQuery) ValidateWritePermission() error {
```

Comment:
After moving test eventContext creation outside storage, this func can be omitted.

Reply:
Function was removed